heal/cli/validate.py

```diff
@@ -22,6 +23,11 @@ def validate(input_file):
     try:
         vlmd_validate(input_file)
         logging.info("Valid")
+    except ValidationError as err:
+        logging.error(
+            f"Error in validating and extracting dictionary from {input_file}"
```

Comment:
```suggestion
            f"Error in validating dictionary from {input_file}"
```
since this function can be called only for validation purpose

Reply:
Nice catch! Yes, this should be distinct from other messages.